myuw/static/vue/components/accounts/tuition-fees.vue

```diff
@@ -280,7 +278,7 @@ export default {
     },
     tuitionDate() {
       // from notice
-      const now = this.nowDatetime();
+      const now = this.nowDatetime(false);
```

Comment:
we may not want to change it to false.

Reply:
false makes it so it never uses the override date